tests/test_2023_10_31.py

```diff
@@ -0,0 +1,657 @@
+    assert has_payments_permission(contracts.acl, contracts.finance, permission.entity, steth["address"], ldo_holder.address, steth["limit"])
+    assert has_payments_permission(contracts.acl, contracts.finance, permission.entity, ldo["address"], ldo_holder.address, ldo["limit"])
+    assert has_payments_permission(contracts.acl, contracts.finance, permission.entity, dai["address"], ldo_holder.address, dai["limit"])
+    assert not has_payments_permission(contracts.acl, contracts.finance, permission.entity, usdt["address"], ldo_holder.address, usdt["limit"])
+    assert not has_payments_permission(contracts.acl, contracts.finance, permission.entity, usdc["address"], ldo_holder.address, usdc["limit"])
+
+    # check regsitries parameters before voting
+    (
+        rcc_already_spent_amount,
+        rcc_spendable_balanceInPeriod,
+        rcc_period_start_timestamp,
+        rcc_period_end_timestamp
+    ) = rcc_stable_registry.getPeriodState()
+
+    assert rcc_already_spent_amount == 800000000000000000000000
+    assert rcc_spendable_balanceInPeriod ==  2200000000000000000000000
+    assert rcc_period_start_timestamp == 1696118400
+    assert rcc_period_end_timestamp == 1704067200
+
+    (
+        pml_already_spent_amount,
+        pml_spendable_balanceInPeriod,
+        pml_period_start_timestamp,
+        pml_period_end_timestamp
+    ) = pml_stable_registry.getPeriodState()
+
+    assert pml_already_spent_amount == 1500000000000000000000000
+    assert pml_spendable_balanceInPeriod ==  4500000000000000000000000
+    assert pml_period_start_timestamp == 1696118400
+    assert pml_period_end_timestamp == 1704067200
+
+    (
+        atc_already_spent_amount,
+        atc_spendable_balanceInPeriod,
+        atc_period_start_timestamp,
+        atc_period_end_timestamp
+    ) = atc_stable_registry.getPeriodState()
+
+    assert atc_already_spent_amount == 800000000000000000000000
+    assert atc_spendable_balanceInPeriod ==  700000000000000000000000
+    assert atc_period_start_timestamp == 1696118400
+    assert atc_period_end_timestamp == 1704067200
+
+    # START VOTE
+    if len(vote_ids_from_env) > 0:
+        (vote_id,) = vote_ids_from_env
+    else:
+        tx_params = {"from": LDO_HOLDER_ADDRESS_FOR_TESTS}
+        vote_id, _ = start_vote(tx_params, silent=True)
+
+    vote_tx = helpers.execute_vote(accounts, vote_id, contracts.voting)
+
+    print(f"voteId = {vote_id}, gasUsed = {vote_tx.gas_used}")
+
+    updated_factories_list = contracts.easy_track.getEVMScriptFactories()
+    assert len(updated_factories_list) == 16
+
+    # check regsitries parameters after voting
+    (
+        rcc_already_spent_amount,
+        rcc_spendable_balanceInPeriod,
+        rcc_period_start_timestamp,
+        rcc_period_end_timestamp
+    ) = rcc_stable_registry.getPeriodState()
+
+    assert rcc_already_spent_amount == 800000000000000000000000
```

Comment:
It seems to me better to check the equality of the parameters before the vote
like:
rcc_already_spent_amount_before == rcc_already_spent_amount_after == 800000000000000000000000